lmdeploy/pytorch/backends/ascend/layer_backend.py

```diff
@@ -1,12 +1,56 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+from typing import Tuple
+
 import torch
 
-from .base_device_utils import BaseDeviceUtils
+from ..base import LayerType
+from ..default import DefaultLayersBackend
+
+
+class AscendLayersBackend(DefaultLayersBackend):
+
+    @staticmethod
+    def get_name() -> str:
+        raise 'ascend'
+
+    @classmethod
+    def get_layer_impl_builder(cls, layer_type: LayerType):
+        if layer_type == LayerType.Attention:
+            from .attention import AscendAttentionImpl
```

Comment:
`AscendAttentionImpl` is not builder, should it be `AscendAttentionBuilder`?

Reply:
Ascend support is not include in this PR. Just ignored.